README.md

```diff
@@ -32,8 +32,16 @@ Stand up the database containers with: `docker-compose up`
 ### docker-compose.admin-test.yml and docker-compose.pupil-test.yml
 Runs SQL Server, MongoDB, all web applications, and the pupil and admin test suites as separate containers.
 This compose file depends on `docker-compose.yml` and should not be run directly.
+
 To stand up the tests, app and db containers run either `./run-compose-admin-test-suite.sh` or `./run-compose-pupil-test-suite.sh`
 
+Both of these scripts require the following values defined in `/admin/.env`:
+```
+APPINSIGHTS_INSTRUMENTATIONKEY='foo-bar'
```

Comment:
how come app insights is no longer optional, am i overlooking something?

Reply:
In my notes, `AZURE_STORAGE_CONNECTION_STRING` and `APPINSIGHTS_INSTRUMENTATIONKEY` values are the same. So I originally had them both coming from the same env var. When you pointed out that they can be different values, I split them out. If `APPINSIGHTS_INSTRUMENTATIONKEY` isn’t required for testing, I can take it out.